common/membership/hashring.go

```diff
@@ -195,12 +193,25 @@ func (r *ring) Subscribe(watcher string, notifyChannel chan<- *ChangedEvent) err
 	return nil
 }
 
-func (r *ring) notifySubscribers(msg *ChangedEvent) {
+func (r *ring) handleUpdates(event ChangedEvent) {
```

Comment:
the parameter is not used, do we plan to use it or we should change the interface?

Reply:
I'd like to keep it maybe for logging.
Not really important, though, you right.